Java/src/com/couchbase/litecore/C4Database.java

```diff
@@ -356,6 +371,10 @@ public static native void copy(String sourcePath, String destinationPath,
 
     static native long nextDocExpiration(long db);
 
+    static native int purgeExpiredDocs(long db);
+
+    static native boolean purgeDoc(long db, String id);
```

Comment:
The signature should be `static native void purgeDoc(long db, String id) throws LiteCoreException;`